config/application.yml.example

```diff
@@ -277,6 +277,7 @@ production:
   piv_cac_agencies: '["DOD"]'
   piv_cac_enabled: 'false'
   pkcs11_lib: '/opt/cloudhsm/lib/libcloudhsm_pkcs11.so'
+  programmable_sms_countries: 'US,CA'
```

Comment:
One suggestion I'd have would be a JSON encoded string here, since we seem the be doing that with these kind of collection elsewhere (see `piv_cac_agencies` above).